### PR TITLE
Add RDS DBCluster W/Storage Encryp Dis query #1057

### LINKED ISSUE #1057: Add RDS DBCluster With Storage Encrypted Disabled query for AWS CloudFormation
### Platform
CloudFormation

### Provider
AWS

### Description
RDS DBCluster should have storage encrypted set to true

### PR DESCRIPTION
**Pull request by joelcarvalhocheckmarx**
RDS DBInstance should have storage encrypted set to true.

Closes #1057

### CODE REVIEW

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Creates RDS Cluster
+Resources:
+  RDSCluster: 
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml:9]: Remove `Ref` to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Resources:
+  RDSCluster: 
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      MasterUserPassword: 
+        Ref: password
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:12]: Remove `Ref` to make this template valid:

`MasterUserPassword: password`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+      MasterUsername: 
+        Ref: username
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:14]: Remove `Ref` to make this template valid:

`MasterUsername: username`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    Properties: 
+      Description: "CloudFormation Sample Aurora Cluster Parameter Group"
+      Family: aurora5.6
+      Parameters: 
+        time_zone: US/Eastern
+    Type: "AWS::RDS::DBClusterParameterGroup"
+  RDSDBInstance1: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml:27]: Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      PubliclyAccessible: "true"
+    Type: "AWS::RDS::DBInstance"
+  RDSDBInstance2: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml:43]: Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      PubliclyAccessible: "true"
+    Type: "AWS::RDS::DBInstance"
+  RDSDBInstance2: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:43]: Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
+      Engine: aurora
+      PubliclyAccessible: "true"
+    Type: "AWS::RDS::DBInstance"
+  RDSDBInstance2_: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:109]: Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+          - Subnet2
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:52]: Remove `Ref` to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Creates RDS Cluster
+Resources:
+  RDSCluster_: 
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:76]: Remove `Ref` to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+          - Subnet1
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml:36]: Remove `Ref `to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+        Fn::GetAtt:
+          - Subnet2
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/negative.yaml:52]: Remove `Ref `to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 12]
```diff
@@ -0,0 +1,130 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Creates RDS Cluster
+Resources:
+  RDSCluster: 
+    Properties: 
+      DBClusterParameterGroupName: 
+        Ref: RDSDBClusterParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:9]: Remove `Ref `to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+        Fn::GetAtt:
+          - Subnet1
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:36]: Remove `Ref `to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:102]: Remove `Ref `to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+        Fn::GetAtt:
+          - Subnet2
+          - AvailabilityZone
+      DBClusterIdentifier: 
+        Ref: RDSCluster
+      DBInstanceClass: db.r3.xlarge
+      DBParameterGroupName: 
+        Ref: RDSDBParameterGroup
+      DBSubnetGroupName: 
+        Ref: DBSubnetGroup
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:118]: Remove `Ref `to make this template valid:

`DBSubnetGroupName: DBSubnetGroup`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+    Properties: 
+      Description: "CloudFormation Sample Aurora Cluster Parameter Group"
+      Family: aurora5.6
+      Parameters: 
+        time_zone: US/Eastern
+    Type: "AWS::RDS::DBClusterParameterGroup"
+  RDSDBInstance1: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:27]: Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`

File: assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml [hunk 17]
```diff
@@ -0,0 +1,130 @@
...
+    Properties: 
+      Description: "CloudFormation Sample Aurora Cluster Parameter Group"
+      Family: aurora5.6
+      Parameters: 
+        time_zone: US/Eastern
+    Type: "AWS::RDS::DBClusterParameterGroup"
+  RDSDBInstance1_: 
+    Properties: 
+      AvailabilityZone: 
+        Fn::GetAtt:
```
Comment (rogeriopeixotocx) [inline assets/queries/cloudFormation/rds_db_cluster_with_storage_encrypted_disabled/test/positive.yaml:93]: Remove `GetAtt` function and leave it hardcoded:

`AvailabilityZone: eu-west-1b`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
Requesting changes

**Comment by rogeriopeixotocx** [comment]
LGTM ✅

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**